src/core.rs

```diff
@@ -371,10 +373,12 @@ impl<T> Webauthn<T> {
         // TODO: Today we send NO EXTENSIONS, so we'll never have a case where the extensions
         // are present! But because extensions are possible from the config we WILL need to manage
         // this situation eventually!!!
+
         match &data.attestation_object.auth_data.extensions {
             Some(_ex) => {
                 // We don't know how to handle client extensions yet!!!
-                return Err(WebauthnError::InvalidExtensions);
+                println!("ext: {:?}", _ex);
+                // return Err(WebauthnError::InvalidExtensions);
```

Comment:
Level 2 doesn't want us to reject all unsolicited or invalid extensions by default anymore. Should we be doing anything else here (left the print and comment as a reminder to myself).

Reply:
And if we do know the extension?

Reply:
The key word is unsolicited. If we are able to parse and handle it, even if unsolicited, we can't reject it.

Reply:
Okay so ignore unknown extensions, and parse known ones. I can also add a field in the extensions output struct that picks up any other extension values so webauthn-rs clients can handle stuff we don't yet support, though perhaps you would prefer people not leaning on that